pkg/storage/allocator_scorer_test.go

```diff
@@ -693,7 +694,7 @@ func TestRebalanceToDiversityScore(t *testing.T) {
 	}
 }
 
-func TestDiversityRemovalScore(t *testing.T) {
+func TestDiversityRemovalAverageScore(t *testing.T) {
```

Comment:
We're going to need a new test (or a restructuring of this test) in order to actually test the kinds of configurations that motivated #17479 in the first place, since this test isn't set up to handle multiple replicas being in the same locality.

Reply:
Yeah, I will add a test for #17479.